openfasoc/generators/ldo-gen/flow/design/sky130hvl/ldo/config.mk

```diff
@@ -53,8 +55,15 @@ export ADDITIONAL_GDS  	        = ../blocks/$(PLATFORM)/gds/capacitor_test_nf.gd
 
 export DOMAIN_INSTS_LIST 	= ../blocks/$(PLATFORM)/ldo_domain_insts.txt
 
+# configuration for routing
+
+export PRE_GLOBAL_ROUTE = ./scripts/openfasoc/pre_global_route.tcl
+
+# informs any short circuits that should be forced during routing
+export CUSTOM_CONNECTION 	= ../blocks/$(PLATFORM)/ldo_custom_net.txt
+
 #export CUSTOM_CONNECTION 	= ../blocks/$(PLATFORM)/tempsenseInst_custom_net.txt
 
-export ADD_NDR_RULE		= 1
-export NDR_RULE_NETS 		= r_VIN
-export NDR_RULE 		= NDR_2W_2S
+#export ADD_NDR_RULE		= 1
```

Comment:
I am guessing you are using the new cmd for NRD. If so then maybe delete these lines.

Reply:
Yes. I will delete this lines